pkg/kube/pod_file_writer.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 The Kanister Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kube
+
+import (
+	"context"
+	"io"
+)
+
+// PodFileRemover provides the mechanism to remove written file from the pod.
+type PodFileRemover interface {
+	Remove(ctx context.Context) error
+	Path() string
+}
+
+// PodFileWriter allows us to write file to POD.
```

Comment:
```suggestion
// PodFileWriter allows us to write files to the pod.
```
[nit] s/POD/pod for consistency